client/src/panels/panel_declarations/finances/auth_exp_planned_spending/auth_exp_planned_spending.js

```diff
@@ -273,10 +273,38 @@ class AuthExpPlannedSpendingGraph extends React.Component {
 }
 
 const render = function ({ calculations, footnotes, sources, glossary_keys }) {
-  const { info, panel_args, subject } = calculations;
+  const { panel_args, subject } = calculations;
   const { data_series, additional_info } = panel_args;
 
-  const final_info = { ...info, ...additional_info };
+  const array_avg = _.chain(data_series)
+    .map((type) => [
+      type.key,
+      _.reduce(
```

Comment:
I think this can just be `_.sum`?

Reply:
whoops i used sum on the bottom half and forgot to go back to fix it!